HelpSource/Classes/SinOsc.schelp

```diff
@@ -314,53 +314,53 @@ Synth.new(\RmOctaver) ;
 
 // midi mapping to some simple stuff using sinosc
 
-mapping sine-waves to MIDI input
+// mapping sine-waves to MIDI input
 
 (
 ~testNote = MIDIFunc.noteOn( { arg ...args;
-	if(args[1] == 60, {~cenaB.value; args.postln;});
-	if(args[1] == 61, {~cenaB.value; ~cenaC1.value; args.postln;});
-	if(args[1] == 62, {~cenaB.value; ~cenaC2.value; args.postln;});
-	if(args[1] == 63, {~cenaB.value; ~cenaC3.value; args.postln;});
-	if(args[1] == 64, {~cenaB.value; ~cenaC4.value; args.postln;});
-	if(args[1] == 65, {~cenaB.value; ~cenaC5.value; args.postln;});
-	if(args[1] == 66, {~cenaB.value; ~cenaC6.value; args.postln;});
-	if(args[1] == 67, {~cenaB.value; ~cenaC7.value; args.postln;});
-	if(args[1] == 68, {~cenaB.value; ~cenaC8.value; args.postln;});
-	if(args[1] == 69, {~cenaB.value; ~cenaC9.value; args.postln;});
+	if(args[1] == 60, {~midiSinB.value; args.postln;});
+	if(args[1] == 61, {~midiSinB.value; ~midiSinC1.value; args.postln;});
+	if(args[1] == 62, {~midiSinB.value; ~midiSinC2.value; args.postln;});
+	if(args[1] == 63, {~midiSinB.value; ~midiSinC3.value; args.postln;});
+	if(args[1] == 64, {~midiSinB.value; ~midiSinC4.value; args.postln;});
+	if(args[1] == 65, {~midiSinB.value; ~midiSinC5.value; args.postln;});
+	if(args[1] == 66, {~midiSinB.value; ~midiSinC6.value; args.postln;});
+	if(args[1] == 67, {~midiSinB.value; ~midiSinC7.value; args.postln;});
+	if(args[1] == 68, {~midiSinB.value; ~midiSinC8.value; args.postln;});
+	if(args[1] == 69, {~midiSinB.value; ~midiSinC9.value; args.postln;});
```

Comment:
I haven't read the rest of this patch, but you might also want to use switch statements - check out the "Control Structures" help file